setup.cfg

```diff
@@ -1,3 +1,48 @@
+[metadata]
+name = build
+version = 0.0.3.1
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Filipe Laíns
+author_email = lains@archlinux.org
+license = MIT
+license_file = LICENSE
+classifiers =
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
3.9?

Reply:
Separate PR where we add 3.9 to the CI too.